packages/backend/src/models/message.ts

```diff
@@ -1,11 +1,11 @@
-import { Document, model, Model } from 'mongoose'
+import { Document, model, Model, QueryOptions } from 'mongoose'
 import { MODEL_MESSAGE } from '@constant/general'
 import { MessageSchema } from '@model/schemas/message'
 
 export interface MessageInterface {
   creator: string
   content: string
-  isVerified: boolean
+  isVerified?: boolean
```

Comment:
I added the question mark because in updateMessage service, I needed the isVerified optional in case the data needed to be updated doesn't include isVerified since if there is no data in isVerified, the db fill it with null. do you have sugestion for this?

Reply:
How about making the `isVerified` in `updateMessageController` required ?

Reply:
What is `isVerified` for? In case a user want to change his/her message's `creator` and `content`, what will the `isVerified` value? Is it the same as before the update?

Reply:
`isVerified` is used to mark messages that are `OK` to be displayed on the website, for the prevention of spam messages and mean content
so if the content is updated I think the message `isVerified` will be set to default false.
But we haven't thought about whether the creator can actually update the message or just a one-time submit. For now i'm assuming it's only for one-time submit

Reply:
Ok, I guess I'll just set `isVerified` to false if it is not found

Reply:
Ok done, should be good now